src/crepe/system/AISystem.cpp

```diff
@@ -13,12 +13,10 @@ using namespace std::chrono;
 void AISystem::update() {
 	const Mediator & mediator = this->mediator;
 	ComponentManager & mgr = mediator.component_manager;
-	LoopTimerManager & timer = mediator.loop_timer;
-	RefVector<AI> ai_components = mgr.get_components_by_type<AI>();
 	LoopTimerManager & loop_timer = mediator.loop_timer;
+	RefVector<AI> ai_components = mgr.get_components_by_type<AI>();
 
-	//TODO: Use fixed loop dt (this is not available at master at the moment)
-	duration_t dt = loop_timer.get_delta_time();
+	float dt = std::chrono::duration<float>(loop_timer.get_scaled_fixed_delta_time()).count();
```

Comment:
ik vind dit niet mooi om hier de fixed delta time omzetten. kan dit niet in de loop timer gedaan worden

Reply:
aangezien er inderdaad veel met floats gewerkt wordt zou je ook in de LooptimerManager de duration_t aanpassen naar een float dan is deze conversie niet nodig